start-chrome-for-playr.sh

```diff
@@ -58,7 +144,21 @@ else
 	reload_url=file://$2
 fi
 
+# Only preference file adjustments are needed on Linux
+if [ "$(uname)" == "Linux" ]; then
```

Comment:
we zullen jhier niet zo zeer het OS maar meer de gebruikte browser moeten checken, Chrome, Chromium(, Edge, Opera, Brave) hebben een dergelijke file maar op een andere locatie, Firefox heeft deze file niet